pkgs/os-specific/darwin/swift-corelibs/corefoundation.nix

```diff
@@ -76,8 +76,8 @@ stdenv.mkDerivation {
   # later.
   buildPhase = stdenv.lib.optionalString true ''
```

Comment:
why? the predicate is literally true, just have the string

Reply:
that was accidental, I initially wanted to make the workaround conditional for clang 7 but that's really hard to check while the stdenv is still bootstrapping.